tests/std/tests/strict_concepts_matrix.lst

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
This could use a comment telling us it's the same as `concepts_matrix.lst` except that all configurations include `/permissive-`.